server/utils/AiProviders/genericOpenAi/index.js

```diff
@@ -94,6 +95,7 @@ class GenericOpenAiLLM {
         model: this.model,
         messages,
         temperature,
+        max_tokens: this.maxTokens,
```

Comment:
We need to spread this property to the `agents` Integration as well to ensure that it wont get blocked by low `max_token` responses as well